### PR TITLE
Merge other regulatory documents into the regulations doc

### PR DESCRIPTION
**Pull request by trubb**
This pull request aims to resolve the issues associated with keeping several separate documents up to date and in accordance with each other and the committee's operations in general.

I am moving the whole pull request as one item, meaning it needs to be accepted in one go.  
The idea is to have any changes to it be in the form of amendments to the pr, in order to make the process easier for all parties.

No english version will be presented until a final approving vote has been made due to the english translations being subordinate to the swedish documents. There is therefore no point in translating all changes before said approval.

_The pull request is part of a motion to the Öråd and may not to be merged without a proper vote being made._  

Merges the following documents into the main regulations document:
1. Styrdokument för Ansvarsposter   
2. Styrdokument för Arbetslag

Updates, clarifies and elaborates the following documents:
1. Prao's Checklista  
2. TraditionsMEsterITs reglemente

### CODE REVIEW

File: svenska/markdown/tmeit_reglemente.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 # Reglemente för TraditionsMEsterIT
 #### Upprättat 2008-12-08
-#### Senast ändrat 2017-11-03
+#### Senast ändrat 2019-XX-XX
```
Comment (trubb) [inline svenska/markdown/tmeit_reglemente.md]: Needs to be changed before merging.

File: svenska/markdown/tmeit_praos_checklista.md [hunk 2]
```diff
@@ -4,19 +4,25 @@
 #### Upprättat 2014-02-13
 #### Senast ändrat 2014-09-13
```
Comment (trubb) [inline svenska/markdown/tmeit_praos_checklista.md]: Needs to be changed before merging.

**Review by trubb — COMMENTED** [review]
Some parts of the documents will need to be updated

File: svenska/markdown/tmeit_praos_checklista.md [hunk 3]
```diff
@@ -4,19 +4,25 @@
...
-- Kan teorin bakom mästeriets märkes-drycker
-- Alkoholtillståndet (Tider, gränser)
-- Alkohollagen
-- Kunskap om TMEITs struktur, organisation och verksamhet. Vraq, Mästare, Marskalk, Prao, Junk/Gourmet/Webb-posterna.
+- Hantering av glaskross och sopor
+- Kassan - Byte av kvittorullar, felslag, återköp m.m.
+- Kvittorapportering av inköp för event
+- Dagsavslut och Z-kvittorapportering efter stängning
+- Inventering, påfyllning, och frontning av bar och spritförråd
+- Skapa stämmning i Kistan 2.0 - musik, belysning, och bildspel
```
Comment (otrapp) [inline svenska/markdown/tmeit_praos_checklista.md]: Passa på att lösa stavfelet på den här raden, stämmning til stämning.

Comment (trubb) [reply svenska/markdown/tmeit_praos_checklista.md]: Fix'd

**Comment by trubb** [comment]
First vote by the Öråd that took place on 2019-01-17 resulted in approval.

**Comment by trubb** [comment]
The second vote by the Öråd that took place on 2019-04-11 resulted in approval, meaning that we can now merge this PR.

**Review by otrapp — APPROVED** [review]
Approving since the Öråd voted through the motion for the second round on 2019-04-11.

**Review by otrapp — APPROVED** [review]
Approving since the Öråd voted through the motion for the second round on 2019-04-11.

**Review by otrapp — APPROVED** [review]
Approving since the Öråd voted through the motion for the second round on 2019-04-11.

**Pull request merged**